tools/static-users-generator/scripts/config_replace.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -e #terminate script immediately in case of errors
+
+TEMPDIR=$(mktemp -d)
+
+cp "/config/src/config.yaml" ${TEMPDIR}
+
+TEMP_FILE_PATH="${TEMPDIR}/config.yaml"
+DEST_FILE_PATH="/config/dst/config.yaml"
+PLACEHOLDER="#__STATIC_PASSWORDS__"
+
+NEWLINE="%"
+
+NUM=0
+for secret in $(kubectl get secrets -l dex-user-config=true --all-namespaces -o json | jq -r -c '.items | .[] | .data')
+do
+
+  EMAIL=$(echo -n ${secret} | jq -r '.email')
+  if [ "${EMAIL}" == "null" ]
+  then
+    continue
+  fi
+
+  EMAIL=$(echo -n ${EMAIL} | base64 -d)
+  
+  USERNAME=$(echo -n ${secret} | jq -r '.username')
+  if [ "${USERNAME}" == "null" ]
+  then
+    continue
+  fi
+
+  USERNAME=$(echo -n ${USERNAME} | base64 -d)
+
+  PASSWORD=$(echo -n ${secret} | jq -r '.password')
+
+  if [ "${PASSWORD}" == "null" ]
+  then
+    continue
+  fi
+
+  PASSWORD=$(echo -n ${PASSWORD} | base64 -d)
+
+  HASH=$(htpasswd -bnBC 10 "" ${PASSWORD} | tr -d ':\n')
+
+  echo "email: ${EMAIL}"
+  echo "username: ${USERNAME}"
+  echo "password: ${PASSWORD}"
+  echo "password encrypted: ${HASH}"
```

Comment:
Do not log passwords :)

Reply:
You are right. I already fixed that in PR #1245, but of course it should be fixed in this PR.